src/features/config/form/birth.ts

```diff
@@ -358,7 +351,7 @@ export const birthRegisterForms: ISerializedForm = {
             exactDateOfBirthUnknown,
             getAgeOfIndividualInYears(
               formMessageDescriptors.ageOfMother,
-              detailsExist
+              exactDateOfBirthUnknownConditional
```

Comment:
Does this conditional include "MOTHER_DETAILS_EXIST" part? Just wanted to be sure

Reply:
that is already in exactDateOfBirthUnknownConditional